app/controllers/solidus_paypal_braintree/client_tokens_controller.rb

```diff
@@ -5,7 +5,7 @@ class ClientTokensController < Spree::Api::BaseController
     before_action :load_gateway
 
     def create
-      render json: { client_token: @gateway.generate_token, payment_method_id: @gateway.id }
+      render json: { client_token: generate_token, payment_method_id: @gateway.id }
```

Comment:
This could be rewritten as

```suggestion
      token = @gateway.generate_token
      if token
        render json: { client_token: generate_token, payment_method_id: @gateway.id }
      else
        render json: { error: Gateway::TOKEN_GENERATION_DISABLED_MESSAGE }, status: 422
      end
```